client/src/modules/MapHolder/components/PopupHolder.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Popup } from 'react-mapbox-gl';
+import PropTypes from 'prop-types';
+import Loader from '../../../shared/Loader';
+
+const useStyles = makeStyles(() => ({
+  popupOuter: {
+    maxWidth: '300px'
+  }
+}));
+
+const PopupHolder = ({ coords, curDefInfo }) => {
+  const classes = useStyles();
+
+  const {
+    _id,
+    __v,
+    //  title,
+    //  address,
+    location,
+    //actual_date,
+    storage_place,
+    accessibility,
+    language,
+    informational_plates,
+    phone,
+    additional_information,
+    ...defInfo
+  } = curDefInfo ? curDefInfo : {};
+
+  return (
+    <>
+      {coords ? (
+        <Popup
+          coordinates={coords}
+          offset={{
+            bottom: [0, -30]
+          }}
+        >
+          {curDefInfo ? (
+            <div className={classes.popupOuter}>
+              {Object.keys(defInfo).map(key => {
+                let field = defInfo[key];
+                if (key === 'actual_date') {
+                  const dateDelim = field.indexOf('T');
+                  const timeDelim = field.indexOf('Z');
+                  field =
+                    field.slice(0, dateDelim) +
+                    ' ' +
+                    field.slice(dateDelim + 1, timeDelim);
+                }
```

Comment:
Maybe use for date format Intl.DateTimeFormat. 
Link to mdn - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat